fr.handbrake.ghb.json

```diff
@@ -0,0 +1,147 @@
+                {
+                    "url": "https://download.handbrake.fr/handbrake/contrib/libdvdnav-5.0.3.tar.bz2", 
+                    "sha256": "5097023e3d2b36944c763f1df707ee06b19dc639b2b68fb30113a5f2cbf60b6d", 
+                    "type": "file", 
+                    "dest-filename": "download/libdvdnav-5.0.3.tar.bz2"
+                }, 
+                {
+                    "url": "https://download.handbrake.fr/handbrake/contrib/libass-0.14.0.tar.gz", 
+                    "sha256": "8d5a5c920b90b70a108007ffcd2289ac652c0e03fc88e6eecefa37df0f2e7fdf", 
+                    "type": "file", 
+                    "dest-filename": "download/libass-0.14.0.tar.gz"
+                }, 
+                {
+                    "url": "https://download.handbrake.fr/contrib/opus-1.2.1.tar.gz", 
+                    "sha256": "cfafd339ccd9c5ef8d6ab15d7e1a412c054bf4cb4ecbbbcc78c12ef2def70732", 
+                    "type": "file", 
+                    "dest-filename": "download/opus-1.2.1.tar.gz"
+                }, 
+                {
+                    "url": "https://download.handbrake.fr/handbrake/contrib/freetype-2.8.1.tar.bz2", 
+                    "sha256": "e5435f02e02d2b87bb8e4efdcaa14b1f78c9cf3ab1ed80f94b6382fb6acc7d78", 
+                    "type": "file", 
+                    "dest-filename": "download/freetype-2.8.1.tar.bz2"
+                }, 
+                {
+                    "url": "https://download.handbrake.fr/handbrake/contrib/libdvdread-5.0.3.tar.bz2", 
+                    "sha256": "321cdf2dbdc83c96572bc583cd27d8c660ddb540ff16672ecb28607d018ed82b", 
+                    "type": "file", 
+                    "dest-filename": "download/libdvdread-5.0.3.tar.bz2"
+                }, 
+                {
+                    "url": "https://download.handbrake.fr/handbrake/contrib/fribidi-0.19.7.tar.gz", 
+                    "sha256": "3fc96fa9473bd31dcb5500bdf1aa78b337ba13eb8c301e7c28923fea982453a8", 
+                    "type": "file", 
+                    "dest-filename": "download/fribidi-0.19.7.tar.gz"
+                }, 
+                {
+                    "url": "https://download.handbrake.fr/handbrake/contrib/harfbuzz-1.7.2.tar.bz2", 
+                    "sha256": "a790585e35c1a87f0dcc23580c84b7cc2324e6f67a2946178d278c2a36c790cb", 
+                    "type": "file", 
+                    "dest-filename": "download/harfbuzz-1.7.2.tar.bz2"
+                }
+            ], 
+            "no-autogen": true, 
+            "builddir": true
+        }
+    ], 
+    "finish-args": [
+        "--share=ipc", 
+        "--socket=x11", 
+        "--socket=wayland", 
+        "--filesystem=xdg-run/dconf", 
+        "--filesystem=~/.config/dconf:ro", 
+        "--talk-name=ca.desrt.dconf", 
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf", 
+        "--filesystem=host"
+    ], 
+    "command": "ghb", 
+    "runtime-version": "3.28", 
+    "runtime": "org.gnome.Platform", 
+    "sdk": "org.gnome.Sdk"
```

Comment:
We try to keep modules after all other fields just for readability.

Reply:
The file is autogenerated.  The order is whatever python json dump writes.

Reply:
If you use an old Python you can put it in an `OrderedDict` (all dicts are ordered in 3.6+)

Reply:
Thanks for the tip.  I'll look into that.